crates/sqlexec/src/planner/logical_plan/mod.rs

```diff
@@ -57,6 +42,17 @@ pub use create_table::*;
 pub use create_temp_table::*;
 pub use create_tunnel::*;
 pub use create_view::*;
+use datafusion::arrow::datatypes::{DataType, Schema as ArrowSchema};
```

Comment:
Idk if rustfmt can do it, but I think having `pub use` seperate from `use` would be preferred. IMO, it's easier to grok what's being exported if all of those are on top. Same applies for `pub mod` vs `mod`

Reply:
I agree with you, and also rustfmt doesn't support it. 😦